influxdb_iox/src/main.rs

```diff
@@ -151,11 +153,12 @@ struct Config {
     )]
     trace_id_header: String,
```

Comment:
```suggestion
    trace_id_headers: Vec<String>,
```

and add a proper default (e.g. two headers) instead of silently hacking in another header if the default is set.

Reply:
Good idea -- will do so

Reply:
in 36bf37b01

Now the help text looks like:

```
      --trace-id-header <TRACE_ID_HEADER>
          HTTP header names sent with Trace ID information
          
          See `--gen-trace-id` to trigger automatic header generation.
          
          [default: uber-trace-id influx-trace-id]
```

And it still works great:

```shell
influxdb_iox -vv --gen-trace-id query NS '<REDACTED>'
2023-05-24T18:14:16.784630Z DEBUG influxdb_iox: Setting trace header name="uber-trace-id" value="c9dda0ba17269e15:1112223334445:0:1"
2023-05-24T18:14:16.784689Z DEBUG influxdb_iox: Setting trace header name="influx-trace-id" value="c9dda0ba17269e15:1112223334445:0:1"
Trace ID set to c9dda0ba17269e15
2023-05-24T18:14:16.785067Z DEBUG hyper::client::connect::http: connecting to 127.0.0.1:8082
2023-05-24T18:14:16.785685Z DEBUG hyper::client::connect::http: connected to 127.0.0.1:8082
2023-05-24T18:14:16.787333Z DEBUG tower::buffer::worker: service.ready=true processing request
... REDACTED ...
Compilation finished at Wed May 24 14:14:18
```